### PR TITLE
Allow responding to reviews

### PR DESCRIPTION
**Pull request by mleeBoeing**
Allow the ability to respond to reviews

- Users should be able to respond to reviews by clicking the chat bubble icon next to the edit icon
- Users can only respond to their own original review multiple times.
- Users can only edit reviews if the user is editing their own response. Otherwise, the edit icon will not show in the first place
- Reset button
    - If creating a new review response, reset button will clear out the text box.
    - If editing an existing response, reset button will revert back the text box to original review response text
- Once you click submit, the text box should automatically remove itself.
- To close the text box, click on the chat bubble icon again.
- Review response text should update automatically if edited.
- Review responses should not reorder upon edit. Should be sorted by created date.
- You cannot submit a response if there are 0 characters in the text area. Submit button is disabled.

### CODE REVIEW

File: app/styles/_quickview.scss [hunk 1]
```diff
@@ -471,3 +479,17 @@ ul.list-unstyled.RecentActivity
         color: $link-color;
     }
 }
+
+.Response {
```
Comment (blynch11p) [inline app/styles/_quickview.scss]: I spoke with @rkenyon1969, go ahead and style the responses to stand out more, please...
```
.Response {
    background: #eeeeef;
    border-left: solid 1px black;
    margin-top: 5px;
    padding: 5px 5px 5px 15px;
}

.list-unstyled.list-reviews ul {
    margin-left: 50px;
}
```

Comment (mleeBoeing) [reply app/styles/_quickview.scss]: Added

**Comment by blynch11p** [comment]
- Need to allow users to respond multiple times to reviews, in conversation form.
- Editing a response should not move it's order in the conversation.

**Comment by mleeBoeing** [comment]
Please review

**Comment by blynch11p** [comment]
- Users should be able to respond to reviews by clicking the chat bubble icon next to the edit icon :+1: 
- Users can only respond to their own original review multiple times. :+1: 
- Users can only edit reviews if the user is editing their own response. Otherwise, the edit icon will not show in the first place :+1: 
- Reset button
    - If creating a new review response, reset button will clear out the text box. :+1: 
    - If editing an existing response, reset button will revert back the text box to original review response text :+1: 
- Once you click submit, the text box should automatically remove itself. :+1: 
- To close the text box, click on the chat bubble icon again. :+1: 
- Review response text should update automatically if edited. :+1: 
- Review responses should not reorder upon edit. Should be sorted by created date. :+1: 
- You cannot submit a response if there are 0 characters in the text area. Submit button is disabled. :+1: 

 :+1:

**Comment by skhtet** [comment]
All the criteria listed in the PR passed :+1:

**Pull request merged**